src/appleseed.cli/CMakeLists.txt

```diff
@@ -59,11 +59,11 @@ add_executable (appleseed.cli
     ${appleseed.cli_sources}
 )
 
-if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
+if (UNIX AND NOT WIN32 AND NOT APPLE)
```

Comment:
I'm fine with `APPLESEED_USE_RPATH`; not so with `APPLESEED_IS_UNIX` as it suggested earlier, since this would introduce conflicting definitions of OS X: a Unix in CMake's definition (and in most everyone's definition), but not a Unix in appleseed's definition...